smartsim/_core/mli/infrastructure/control/workermanager.py

```diff
@@ -210,49 +115,88 @@ def __init__(
 
         self._task_queue: t.Optional[CommChannelBase] = config_loader.get_queue()
         """the queue the manager monitors for new tasks"""
-        self._feature_store: t.Optional[FeatureStore] = (
-            config_loader.get_feature_store()
-        )
-        """a feature store to retrieve models from"""
         self._worker = worker
         """The ML Worker implementation"""
-        self._comm_channel_type = comm_channel_type
+        self._callback_factory = config_loader._callback_factory
         """The type of communication channel to construct for callbacks"""
         self._device = device
         """Device on which workers need to run"""
         self._cached_models: dict[str, t.Any] = {}
         """Dictionary of previously loaded models"""
+        self._feature_stores: t.Dict[str, FeatureStore] = {}
+        """A collection of attached feature stores"""
+        self._fs_factory = config_loader._featurestore_factory
+        """A factory method to create a desired feature store client type"""
+        self._backbone: t.Optional[FeatureStore] = config_loader.get_backbone()
+        """The backbone feature store"""
```

Comment:
I know I wrote this somewhere else, but I definitely think more detail about what a backbone feature store is would be nice! "Backbone" is kind of descriptive, I guess? But I could use more.

Reply:
it might become redundant if we add a `class BackboneFS` later, but I think I have improved by replacing with:
`A standalone, system-created feature store used to share internal information among MLI components`

Reply:
Love it!